answer/scanQRAck.json

```diff
@@ -0,0 +1,25 @@
+{
+	"$schema": "http://json-schema.org/draft/2019-09/schema#",
+	"$id": "https://github.com/dedis/student20_pop/tree/proto-specs/answer/scanQRAck.json",
+	"title": "Match a custom JsonRpc 2.0 message answer",
+	"description": "Match a scan QR acknowledgement",
+	"type": "object",
+
+	"properties": {
+		"jsonrpc": { "description": "[String] JsonRpc version", "const": "2.0", "$comment": "should always be \"2.0\"" },
+		"address": { "description": "[String] address of server", "const": "2.0", "$comment": "Organizer's back end server address" },
```

Comment:
I'm quite sure this is not valid JSON RPC

Reply:
See this [link](https://www.jsonrpc.org/specification) guys. The problem is that (for example) `address` should be located inside the `result` object

Reply:
If you change the file to conform to json rpc, you should be able to add one `$ref` in the file `answer/answer.json`  and define this object (`scanQRAck`) inside the `positiveAnswer` folder

If, for some reason, this cannot be done and you have to keep this file directly in the `answer` folder, then you will also need to modify the top-level file to include a `$ref` to this file